test/walkUp.js

```diff
@@ -0,0 +1,26 @@
+import walkUp from "../src/walkUp.js";
+import trees from "./utils/trees.js";
+import updateParents from "../src/updateParents.js";
+import copy from "./utils/copy.js";
+
+const tree = copy(trees[0]);
+updateParents(tree);
```

Comment:
Might want to have some `throws` tests too, for when we don't have parents. Or should that just be handled in the parents tests?